share/src/main/java/io/zensoft/share/service/facebook/FacebookPublisherManagerService.java

```diff
@@ -31,37 +30,39 @@ public FacebookPublisherManagerService(DefaultVacancyConverterService vacancyCon
                                            DefaultVacancyModelService vacancyModelService,
                                            DefaultVacancyResponseModelService vacancyResponseModelService,
                                            PublisherService facebookPublisherService,
+                                           VacancyRetrieverService defaultVacancyRetrieverService,
                                            VacancyResponseSenderService vacancyResponseSenderService) {
         this.vacancyConverterService = vacancyConverterService;
         this.vacancyResponseConverterService = vacancyResponseConverterService;
         this.vacancyModelService = vacancyModelService;
         this.vacancyResponseModelService = vacancyResponseModelService;
         this.facebookPublisherService = facebookPublisherService;
+        this.defaultVacancyRetrieverService = defaultVacancyRetrieverService;
         this.vacancyResponseSenderService = vacancyResponseSenderService;
     }
 
     @Override
     public void publish(VacancyDto vacancyDto) {
         Vacancy vacancy = convertToVacancyAndSaveToDatabase(vacancyDto);
         VacancyResponse vacancyResponse = facebookPublisherService.publish(vacancy);
-        saveToDatabaseAndConvertToDtoAndRespond(vacancyResponse);
+        vacancyResponseModelService.save(vacancyResponse);
+        convertToDtoAndRespond(vacancyResponse);
     }
 
     @Override
-    public void getInfo(VacancyDto vacancyDto) {
+    public void getInfo(VacancyDto vacancyDto, PublisherServiceType publisherServiceType) {
```

Comment:
i think would be better creating `PublisherServiceType` object in the body of the method, instead of taking it as an argument, cause i don't think that it's a good idea to take it from `FacebookListener`